docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+
+###
+# Get a list of applications
+#
+get:
+  description: Get the list of running applications. Several filters can be applied to filter the result list.
+  is: [ secured ]
+  queryParameters:
+    cmd:
+      description: the command of this application
+      example: java
+    id:
+      description: the id or part of the id of the application
+      example: /us-east/database/memsql
+    label:
+      description:
+        A label selector query contains one or more label selectors, which are comma separated.
+        Marathon supports three types of selectors existence-based, equality-based and set-based.
+        In the case of multiple selectors, all must be satisfied so comma separator acts as an AND logical operator.
+        Labels and values must consist of alphanumeric characters plus `-` `_` and `.` `-A-Za-z0-9_.`.
+        Any other character is possible, but must be escaped with a backslash character.
+
+        * <code>Existence based Selector Query</code> Matches the existence of a label
+
+        * <code>Equality based Selector Query</code> Matches existence of labels and the (non) equality of the value.
+
+        * <code>Set based Selector Query</code> Matches existence of labels and the (non) existence of the value in a given set
+
+      example: environment==production, tier!=frontend\ tier, deployed in (us, eu), deployed notin (aa, bb)
+    embed:
+      description:
+        Embeds nested resources that match the supplied path.
```

Comment:
I figure this is copied from the former docs, but in general, I think it'd be easier to read if all flags followed an imperative (_eek_) style along with italic annotations, like 
`"apps.counts"`: embed task counts (tasksStaged, tasksRunning, tasksHealthy, tasksUnhealthy). _Note: these are currently embedded by default but this will change in a future release._